converter/ena/base.py

```diff
@@ -50,7 +52,9 @@ def _get_entity_content(entity: dict) -> dict:
     def _get_value_by_key_path(entity: dict, key_path: list) -> str:
```

Comment:
This function could be in a separate module. It is not specific to ena.

Reply:
Moved all of these kind of utility methods for conversion to package level